proto/penumbra/penumbra/core/component/shielded_pool/v1/shielded_pool.proto

```diff
@@ -97,6 +110,12 @@ message EventOutput {
   crypto.tct.v1.StateCommitment note_commitment = 1;
 }
 
+// ABCI Event recording a clue.
+message EventClue {
```

Comment:
tioli: I think it would read better if there was a noun or a verb to anchor some context e.g. `EventBroadcastClue` or `EventTransactionClue`